source/base/function_lib.cc

```diff
@@ -2081,9 +2080,9 @@ namespace Functions
   template <int dim>
   double
   FourierSineSum<dim>::laplacian(const Point<dim>  &p,
-                                 const unsigned int component) const
+                                 [[maybe_unused]] const unsigned int component) const
   {
-    (void)component;
+    
     AssertIndexRange(component, 1);
```

Comment:
Here and in the other places, there is really no good reason to leave an empty line just because there used to be something there. Could you just remove the empty lines?